web_src/js/features/repo-legacy.js

```diff
@@ -139,7 +139,7 @@ export function initRepoCommentForm() {
 
       hasUpdateAction = $listMenu.data('action') === 'update'; // Update the var
 
-      const $clickedItem = $(this);
+      const clickedItem = this; // eslint-disable-line unicorn/no-this-assignment
```

Comment:
This function has a major headache with multiple `this` scopes, it will require a deeper refactor to avoid this, so I went with the simple route for now.